techniques/system/common/1.0/promises.st

```diff
@@ -21,21 +21,44 @@
 # promises.cf
 #
 #######################################################
-body common control
+bundle common rudder_roles
 {
+  # variables are defined in bundle common g
+  classes:
+      # Policy Server is a machine which delivers promises
+      "policy_server" expression => strcmp("root","${g.uuid}");
+      # Root Server is the top policy server machine
+      "root_server" expression => strcmp("root","${g.uuid}");
+
+
+      # on changes only, we only reports on repaired or error message
+      "changes_only"     expression => strcmp("&RUDDER_REPORT_MODE&", "changes-only");
+      "reports_disabled" expression => strcmp("&RUDDER_REPORT_MODE&", "reports-disabled");
 
+      # full compliance is the default mode
+      "full_compliance" not => "changes_only|reports_disabled";
+}
+
+body common control
+{
     any::
         output_prefix => "rudder";
         
         protocol_version   => "classic";
 
         inputs => {
-          @{va.ncf_inputs},
+          @{va.inputs_list},
           "rudder_system_directives.cf",
           "rudder_directives.cf"
         };
 
-        bundlesequence => { @{va.bs}, rudder_system_directives, rudder_directives, @{va.end} };
+        bundlesequence => { 
+          @{va.common_bs},
+          @{va.specific_bs},
+          rudder_system_directives,
+          rudder_directives,
+          @{va.end}
+        };
 
     android::
         # if native syslog is not available, use cfengine implementation (eg: android)
```

Comment:
This block didn't use to be in initial promises, and since initial promises shouldn't send reports, I think that it still shouldn't be, because I believe this causes CFEngine to send messages to syslog. Could you surround it with if(INITIAL)?